source/blog/2017-12-02-installing-etcd-as-a-system-container.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Installing etcd as a System container
+author: smilner
+date: 2017-10-20 20:00:00 UTC
+published: false
+comments: true
+tags: atomic, etcd, system containers, runc, containers
+---
+
+System Containers exist as a way to provide containerized services to a host before traditional container runtimes are ready. System
+Containers do this by utilizing a runc and systemd units and do not deviate from the OCI standards. Let's look at how someone can
+use a system containers to install the ever popular etcd today one Fedora 27 Atomic Host!
+
+READMORE
+
+
+## Atomic Host
+
+The first thing one must have is Fedora 27 Atomic Host installed. As a reminder, Atomic Host allows people to:
+
+> Use immutable infrastructure to deploy and scale your containerized applications. Project Atomic builds OSes, tools, and containers for cloud native platforms.
```

Comment:
hey, since this blog post is going onto the website that quote is from, we probably don't need it.  Appreciate the messaging, though.